tutorbackup/patches/k8s-jobs

```diff
@@ -59,11 +59,43 @@ spec:
               value: '{{ BACKUP_S3_SECRET_ACCESS_KEY }}'
             - name: S3_BUCKET_NAME
               value: '{{ BACKUP_S3_BUCKET_NAME }}'
-      {% if ENABLE_WEB_PROXY %}
+          {% if BACKUP_K8S_USE_EPHEMERAL_VOLUMES or ENABLE_WEB_PROXY %}
           volumeMounts:
+          {% endif %}
+            {% if BACKUP_K8S_USE_EPHEMERAL_VOLUMES %}
+            - mountPath: /data/
```

Comment:
I get what the `data-volume` is for, but can you please explain the rationale behind a separate additional `backup-volume`? What's the benefit of creating *two* ephemeral volumes, as opposed to modify the backup scripts so that `TARFILE` also lives in `/data` (perhaps in `/data/backup` or some such)?

Reply:
Yes, you can change [this line](https://github.com/hastexo/tutor-contrib-backup/blob/afb824f82d803d8e035aa53e259ec6053251a712/tutorbackup/templates/backup/build/backup/backup_services.py#L25) to put the tarfile in `/data/backup/`. Then you won't need `backup-volume`.

Reply:
> Yes, it would be better to have everything in one single volume mount. However we would need to allocate double space, for the tarfile and the uncompressed files.

I *think* the Python `tarfile` module does write out an xz-compressed data stream. Which means it really shouldn't be duplicating all of the space usage.

> I think it would be easier to remove the /data volume and use /backup/data as the dump directory. I'll prepare these changes

How exactly does that make things easier? Sounds rather like a toss-up to me.

Reply:
The problem with that is that in [local-docker-compose-jobs-services](https://github.com/hastexo/tutor-contrib-backup/blob/afb824f82d803d8e035aa53e259ec6053251a712/tutorbackup/patches/local-docker-compose-jobs-services#L20), `/backup/` is mapped to the `<tutor-config-root>/env/backup/` on the host machine. That's where the tarfile is stored when using `tutor local backup`. If you move the dump directory to `backup/data/` then the dumps will also be stored there (on user's local disk). Unless you clean them up after creating the tarfile, of course.

However, making the `/backup` a subdirectory of `/data/` avoids that. Just remember to update the mapping in both the script and in local-docker-compose-jobs-services.

Reply:
> The problem with that is that in [local-docker-compose-jobs-services](https://github.com/hastexo/tutor-contrib-backup/blob/afb824f82d803d8e035aa53e259ec6053251a712/tutorbackup/patches/local-docker-compose-jobs-services#L20)

This is what was confusing me... Ok, I'll do as you suggest and test again

Reply:
@angonz besides remapping the backup directory path, and updating the commit message, please rebase this PR as well. Then I think this will be good to merge.